glances/.env-dist

```diff
@@ -0,0 +1,51 @@
+# The docker image to use:
+GLANCES_IMAGE=nicolargo/glances:4.1.2.1
+
+# The domain name for the glances service:
+GLANCES_TRAEFIK_HOST=glances.example.com
+
+# The name of this instance. If there is only one instance, use 'default'.
+GLANCES_INSTANCE=
+
+# Filter access by IP address source range (CIDR):
+##Disallow all access: 0.0.0.0/32
+##Allow all access: 0.0.0.0/0
+GLANCES_IP_SOURCERANGE=0.0.0.0/0
+
+# HTTP Basic Authentication:
+# Use `make config` to fill this in properly, or set this to blank to disable.
+GLANCES_HTTP_AUTH=
+
+# OAUTH2
+# Set to `true` to use OpenID/OAuth2 authentication via the
+# traefik-forward-auth service in d.rymcg.tech.
+# Using OpenID/OAuth2 will require login to access your app,
+# but it will not affect what a successfully logged-in person can do in your
+# app. If your app has built-in authentication and can check the user
+# header that traefik-forward-auth sends, then your app can limit what the
+# logged-in person can do in the app. But if your app can't check the user
+# header, or if your app doesn't have built-in authentication at all, then
+# any person with an account on your Gitea server can log into your app and
+# have full access.
+GLANCES_OAUTH2=
+# In addition to Oauth2 authentication, you can configure basic authorization
+# by entering which authorization group can log into your app. You create
+# groups of email addresses in the `traefik` folder by running `make groups`. 
+GLANCES_OAUTH2_AUTHORIZED_GROUP=
+
+# Mutual TLS (mTLS):
+# Set true or false. If true, all clients must present a certificate signed by Step-CA:
+GLANCES_MTLS_AUTH=false
+# Enter a comma separated list of client domains allowed to connect via mTLS.
+# Wildcards are allowed and encouraged on a per-app basis:
+GLANCES_MTLS_AUTHORIZED_CERTS=*.clients.glances.example.com
+
+# Timezone
+GLANCES_TZ=EST5EDT
```

Comment:
Does it support timezones like `America/New_York`? I think that would be a more intuitive example.

Reply:
link in the comments if appropriate: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Reply:
Fixed in [e86bce8](https://github.com/EnigmaCurry/d.rymcg.tech/pull/284/commits/e86bce8c4914bb96cc0a5d4f9d7427fc8c3c1ac5)